site/content/docs/v1.16/repository-maintenance.md

```diff
@@ -155,9 +155,22 @@ Status:
 - `Recent Maintenance` keeps the status of the recent 3 maintenance jobs, including its start time, result (succeeded/failed), completion time (if the maintenance job succeeded), or error message (if the maintenance failed)
 
 ### Others
-Maintenance jobs will inherit the labels, annotations, toleration, nodeSelector, service account, image, environment variables, cloud-credentials etc. from Velero deployment.  
+Maintenance jobs will inherit specific labels, annotations, toleration, nodeSelector, service account, image, environment variables, cloud-credentials etc. from Velero deployment.
```

Comment:
To address the confusion mentioned in issue #8974, we should make it clear that labels and annotations are not inherited from the Velero deployment directly anymore.